module/documents/activity/order.mjs

```diff
@@ -48,6 +48,19 @@ export default class OrderActivity extends ActivityMixin(OrderActivityData) {
     }
   }, { inplace: false }));
 
+  /* -------------------------------------------- */
+  /*  Properties                                  */
+  /* -------------------------------------------- */
+
+  /** @inheritDoc */
+  get canUse() {
+    return super.canUse
+      // Don't allow usage if facility is already executing the same order
+      && (this.parent.progress.order !== this.order)
```

Comment:
#4713 is not really a bug. It's intentional that you can re-issue the same order in case you made a mistake and want to change some parameter of it. We can probably disallow it if `progress.value > 0`, and anyone who wants to change their order at that point has to manually edit the facility to reset progress.